libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/extension/AuditExtensionsReceivedVcEvidence.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.di.ipv.core.library.auditing.extension;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import lombok.Getter;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+@ExcludeFromGeneratedCoverageReport
+@Getter
+public class AuditExtensionsReceivedVcEvidence extends AuditExtensionsVcEvidence {
```

Comment:
AuditExtensionsReceivedVcEvidence (renamed from AuditExtensionsVcEvidence) extends the new AuditExtensionsVcEvidence because it's specific to received vcs. The new AuditExtensionsVcEvidence can then be used for the extension to the IPV_VC_REVOKED audit event

Reply:
Is it specific just because of the `successful` field?

One other option would be to make the `successful` property a `Boolean` (capital B) and make it `@JsonInclude(NON_NULL)` - that way it'll be omitted if set to null.

See `AuditExtensionsReproveIdentity` for an example